bitwarden_license/src/app/main.ts

```diff
@@ -7,6 +7,7 @@ import "popper.js";
 
 // tslint:disable-next-line
 require("src/scss/styles.scss");
+require("src/scss/tailwind.css");
```

Comment:
Adding tailwind as a separate `css` file prevents sass from having to re-compile all the time. And cuts down the compile time from 20s to 1s.